script.js

```diff
@@ -2,83 +2,110 @@
  * Section
  * ======================================================================== */
 
-var orders = [];
-
 document.addEventListener('DOMContentLoaded', function() {
 
-  // get orders from page
+      // get orders from page
 
-  chrome.tabs.executeScript(null, {
-    code: "\
-      var orders = [];\
-      if (document.querySelectorAll('h1')[0].innerText.toLowerCase().indexOf('istoric comenzi') != -1) {\
-        var tags = document.querySelectorAll('.history-item')[0].querySelectorAll('.content');\
-        for (var i = 0; i < tags.length; i++) {\
-          orders.push({\
-            name: tags[i].querySelector('h4').innerText.trim(),\
-            price: tags[i].querySelector('.price').innerText.trim(),\
-          });\
+    chrome.tabs.executeScript(null, {
+        code: "\
+      var orders = [], iterator, tags, dinerNameDOMElement, dinerName, priceValueDOMElement, priceValue;\
```

Comment:
`iterator` usually refers to an object that helps you iterate over a collection. Replacing the `i - index` with `iterator` is a bad idea. Please replace `iterator` with `i`.